src/js/components/Friends/FriendDisplayForList.jsx

```diff
@@ -88,17 +88,15 @@ class FriendDisplayForList extends Component {
   }
 }
 FriendDisplayForList.propTypes = {
-  linked_organization_we_vote_id: PropTypes.string,
-  mutual_friends: PropTypes.number,
-  positions_taken: PropTypes.number,
+  linkedOrganizationWeVoteId: PropTypes.string,
```

Comment:
@QueMona98 I'm glad you updated these incoming variables to camelCase here, but you will also need to make the updates throughout the file, like around line 12:
```
    const {
      mutual_friends: mutualFriends,
      positions_taken: positionsTaken,
      voter_we_vote_id: voterWeVoteId,
      voter_photo_url_large: voterPhotoUrlLarge,
    } = this.props;
```